R/separate.R

```diff
@@ -158,3 +158,15 @@ list_indices <- function(x, max = 20) {
 
   paste(x, collapse = ", ")
 }
+
+strpieces <- function(value, sep, n_max, extra) {
+  pieces <- strsplit(value, sep)[[1]]
```

Comment:
Is it possible to vectorise this function? Otherwise this might end up being a major performance hit.

Reply:
Updated